tensorbay/cli/auth.py

```diff
@@ -5,17 +5,18 @@
 
 """Implementation of gas auth."""
 
-
 from configparser import ConfigParser
 from textwrap import indent
 from typing import Dict, Optional
 from urllib.parse import urljoin
 
 import click
 
+from ..exception import ResponseSystemError
```

Comment:
this commit type should be `feat` rather than `refactor`
also this commit message is not clear, needs to be refined.